src/conn/conn_log.c

```diff
@@ -986,10 +984,8 @@ __wt_logmgr_create(WT_SESSION_IMPL *session)
     WT_RET(__wt_spin_init(session, &log->log_sync_lock, "log sync"));
     WT_RET(__wt_spin_init(session, &log->log_writelsn_lock, "log write LSN"));
     WT_RET(__wt_rwlock_init(session, &log->log_remove_lock));
-    if (FLD_ISSET(conn->direct_io, WT_DIRECT_IO_LOG))
-        log->allocsize = (uint32_t)WT_MAX(conn->buffer_alignment, WT_LOG_ALIGN);
-    else
-        log->allocsize = WT_LOG_ALIGN;
+
+    log->allocsize = WT_LOG_ALIGN;
```

Comment:
Do you think it's worth getting rid of `log->allocsize` and completely replacing it with the macro?